media-tv/plex-media-server/plex-media-server-1.22.2.4282.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+MY_COMMIT="a97b03fad"
```

Comment:
My personal preference is to put MY_* variables above DESCRIPTION, then have no whitespace between DESCRIPTION/HOMEPAGE/SRC_URI/S, then a newline, then LICENSE and friends. Up to you though.